docs/components/camera/webcam.md

```diff
@@ -143,9 +143,14 @@ If this doesn't work, you can reboot your machine by running:
 sudo reboot
 ```
 
+### Images are dim on start up
+
+If you are capturing camera data, it can happen that the camera captures and syncs miscolored or dark images upon start up.
```

Comment:
Maybe just add that this is part of the normal calibration process for a camera?